### ISSUE TITLE
Multi Object Property Editing / Prefabs

### ISSUE THREAD

**Issue by MR-Alex42** [Frooxius/NeosPublic#83]
Currently you need to apply a change of properties to each and every object separately. It would be much more efficient if a user could organize objects into groups and apply a change to all members of the group. Another way to achieve this would be to introduce a concept like Unity prefabs. If the prefab changes all instances of the prefab change as well.

**Comment by sirkitree** [comment]
> For example, should the prefabs be applied across all worlds?

Please no. I think being able to copy things to their own worlds is a huge benefit of Neos, and if someone changes the original to a penis... well, you can imagine the consequences.

**Comment by Frooxius** [comment]
@sirkitree Yeah that's a good point :D Although I imagine both workflows would be necessary. Say you're making a game that's composed of dozens of worlds and you want to update an enemy that's in most of them. You'd want those changes to apply across your experiences.

There would be a way to break the reference to the original or create your own duplicate of it. But it should be designed in a way that makes it clear which one is happening with some sensible defaults.

**Comment by sirkitree** [comment]
Maybe just by author then?

If I import an asset, and use it in multiple worlds, only I can replace it across all of my worlds.

If someone copies that asset and uses it in their own worlds, they could update it across their worlds, but I cannot update that same asset that they copied into their worlds.
